frontend/src/plugin/__snapshots__/pluginLib.snapshot

```diff
@@ -16100,10 +16100,6 @@
   "Utils": {
     "CLUSTER_ACTION_GRACE_PERIOD": 5000,
     "auth": {
-      "deleteTokens": [Function],
-      "getToken": [Function],
```

Comment:
Some plugins use token related functions.

Reply:
they'll keep working, but only if plugins provide their own get/setToken functions via registerGetTokenFunction, registerSetTokenFunction

Reply:
otherwise, it'll stop working since js can't access tokens anymore, which is the point